modules/efk-logging-elasticsearch-ha.adoc

```diff
@@ -20,6 +20,10 @@ Logs might be unavailable or lost in the event a node is down or fails.
 Use this mode when you are more concerned with performance than safety, or have
 implemented your own disk/PVC backup/restore strategy.
 
+[NOTE]
+====
+The number of primary shards is always equals to the number of Elasticsearch nodes.
```

Comment:
I think we want to be clear that this is specifically for our index templates. The number of primary shards on an index is set during creation and cannot be changed after that... also specifically data nodes.

"The number of primary shards for the index templates is equal to the number of Elasticsearch data nodes".